sycl/include/sycl/detail/array.hpp

```diff
@@ -9,8 +9,8 @@
 #pragma once
 
 #include <sycl/detail/defines_elementary.hpp> // for __SYCL_ALWAYS_INLINE
-#include <sycl/detail/pi.h>                   // for PI_ERROR_INVALID_VALUE
-#include <sycl/exception.hpp>                 // for invalid_parameter_error
+#include <sycl/detail/exception_helper.hpp>
```

Comment:
Nit; Please specify what we are using in the include in a comment, like the other includes.

Reply:
We had some discussion about these.
We actually decided not to do this kind of comments because it is not very useful and hard to maintain.